libs/ui/src/components/atoms/Badge/Badge.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode, RefObject, forwardRef } from 'react';
+
+import { Bold, ParMd, ParXs } from '../Typography';
+
+import { BaseBadge } from './Badge.styles';
+
+export type BadgeProps = {
+  badgeLabel: ReactNode;
+  badgeSize: 'sm' | 'lg';
+  badgeColor: 'blue' | 'green' | 'pink' | 'violet';
+  className?: string;
+};
+
+type Ref =
+  | ((instance: HTMLBaseElement | null) => void)
+  | RefObject<HTMLBaseElement>
+  | null
+  | undefined;
+
+export const Badge = forwardRef(
+  (
+    {
+      className,
+      badgeLabel,
+      badgeSize = 'sm',
+      badgeColor = 'blue',
+    }: BadgeProps,
+    ref: Ref
+  ) => {
+    return (
+      <BaseBadge
+        badgeColor={badgeColor || 'blue'}
+        badgeSize={badgeSize || 'sm'}
```

Comment:
Line 25 and 26 are esentially doing the same thing.

25, 26 you set a default values for the props.
32, 33 you are checking if the badgeColor or badgeSize is truthy and if not setting it to a value.

Reply:
updating to just one.